scripts/deploy-blobstreamx.sh

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
+source ../.env
```

Comment:
As `starkli` uses these same environment variables the dev may have them in the env already. Lets parse this `.env` only if the environment variables aren't already set.

Reply:
This will also make the script less path dependent

Reply:
Made the change